.travis.yml

```diff
@@ -20,6 +20,8 @@ stages:
 
 import:
   - source: Alfresco/alfresco-process-tools:.travis.kubernetes_install.yml@master
+  - source: Alfresco/alfresco-process-tools:.travis.helm.yml@master
```

Comment:
Doing this breaks our build, see https://travis-ci.com/github/Alfresco/acs-deployment/builds/191249066 so it was omitted previously purposely. 

The publish task runs too early. I raised a support ticket with Travis previously, there is a way to fix the order but we have also customised the publish process slightly so can't use the generic one.

If we can decide on a common way of publishing charts we might be able to fix it.

Reply:
got it wrong sorry, it should have been just install